src/Features/ERDDAP/hooks/tabledap.ts

```diff
@@ -16,7 +16,7 @@ import { defaultQueryConfig } from "./hookConfig"
 
 const FORBIDDEN = "Forbidden"
 
-export const getDataset = (timeSeries: PlatformTimeSeries, startTime?: Date) => {
+export const getDataset = (timeSeries: PlatformTimeSeries, startTime?: Date, endTime?: Date) => {
   return async () => {
     const url = urlBuilder([timeSeries], startTime)
```

Comment:
The `endTime` likely needs to make it's way into `urlBuilder`, and for `urlBuilder` to also know what it can do with that.

Reply:
Yes, thank you! I was mid way through trying to add end time in when I initially pushed that up. 

Because Tom is wanting to be able to see the changes in the dev server, I think its not in the scope of this PR (i.e. I don't think I will have time) to add the predicted tides in the future. However, I did add some changes so the skeleton of being able to see future predicted tides is built out--just not functional yet. I've updated the current screenshot.